frontend/src/components/@common/Toggle/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { useCallback } from 'react';
+import { Circle, ToggleBtn, Wrapper } from './Toggle.style';
+
+interface ToggleProps {
+  width?: number;
+  height?: number;
+  state: boolean;
+  toggleOnCallback?: () => void;
+  toggleOffCallback?: () => void;
```

Comment:
onChange 로 통일해서 `!state`를 넘겨주는 interface는 어떤지 제안해봅니다!
```suggestion
  onChange?: (state: boolean) => void;
```

```
<ToggleBtn
        onClick={() => { onChange?.(!state) }}
```

Reply:
현재 toggle에 넘기는 state가 서버에서 받아온 상태인데 그 상태를 임의로 !해서 넘기는 것은 잘 모르겠네요 ㅠㅠ. 저도 Toggle을 작성하면서 범용성이 있어보이지는 않아서 고민이긴 합니다.

현재 구독 정보를 반환하는 메서드에 낙관적 업데이트를 적용해놔서 저희가 직접적으로 상태를 변경시키는 것보다는 인자로 넘어온 것을 사용하는게 좋다고 생각합니다!

Reply:
현재 toggle에 넘기는 state가 서버에서 받아온 상태인데 그 상태를 임의로 !해서 넘기는 것은 잘 모르겠네요 ㅠㅠ. 저도 Toggle을 작성하면서 범용성이 있어보이지는 않아서 고민이긴 합니다.

현재 구독 정보를 반환하는 메서드에 낙관적 업데이트를 적용해놔서 저희가 직접적으로 상태를 변경시키는 것보다는 인자로 넘어온 것을 사용하는게 좋다고 생각합니다!

Reply:
오 현재 낙관적 업데이트긴 하네요
장바구니 미션의 개수 버튼처럼 토글 클릭 한번에 로딩과 에러를 다 핸들링하는 방향은 어떤가요?

Reply:
현재 에러는 핸들링 하네요! 꼼꼼하게 못봐서 죄송합니다ㅠ
실패하는 일이 크게 일어날 것 같진 않아서 이대로도 괜찮을 것 같아요!